packages/ui-components/src/components/Map/utils/markerFormats.js

```diff
@@ -190,13 +190,36 @@ export function flattenMeasureHierarchy(measureHierarchy) {
   return results;
 }
 
-function getValueInfo(value, valueMapping, hiddenValues = {}) {
+const getIsHidden = (measureData, serieses, allHiddenValues) =>
+  serieses
+    .map(({ key, valueMapping, hideByDefault }) => {
```

Comment:
Most of the code in the block below this is somewhat repeating the logic in `getValueInfo`...but it's just different enough that after staring for a while it's hard to see how to nicely DRY it out. I'm happy to approve it, but I reckon if the rest of this file got a refactor at some point it could all be cleaned up a fair bit.